code/script.js

```diff
@@ -1,53 +1,292 @@
 // DOM selectors (variables that point to selected DOM elements) goes here 👇
-const chat = document.getElementById('chat')
+const chat = document.getElementById("chat");
+const nameInput = document.getElementById("name-input");
+const nameForm = document.getElementById("name-form");
+const inputWrapper = document.getElementById("input-wrapper");
+
+// Global variables
+let userName = "";
+let animalChoice = "";
+let selectedType = "";
+let gender = "";
+let animalName = "";
+let confirmApp = "";
 
 // Functions goes here 👇
+// This function inserts the animal option buttons into the html
+const addAnimalButtons = () => {
+  inputWrapper.innerHTML = `
+    <button class="send-btn" id="cat">Cat</button>
+    <button class="send-btn" id="dog">Dog</button>
+    <button class="send-btn" id="horse">Horse</button>`;
+};
```

Comment:
Nice that you've broken out these into their own function. Thought: Do you think it would be possible to refactor this even more? Nothing you have to do, but just a thought experiment ;) Example:

```
const animalButtons = `
    <button class="send-btn" id="cat">Cat</button>
    <button class="send-btn" id="dog">Dog</button>
    <button class="send-btn" id="horse">Horse</button>`;

const changeHTML = (param) => {
  if (some condition) {
    inputWrapper.innerHTML = animalButtons
  } else if (some other condition) {
    //show other HTML elements
  }
}
```